tools/cli.js

```diff
@@ -29,7 +29,14 @@ function complete(input, completions) {
   return completions.filter(c => c.startsWith(input));
 }
 
-function tryComplete(input, completer) {
+function tryComplete(input, completions) {
+  if (!completions) return input;
+  completions = complete(input, completions);
+  if (completions.length === 1) return completions[0];
+  return input;
+}
+
+function tryCompleter(input, completer) {
```

Comment:
It's more of a generic question. As far as I can see, there are functions named `complete()` and `completer()` in master already, so I'd say go ahead with `tryCompleter()` in this PR for the sake of consistency. We'll need to come up with better names for all four functions in another PR though.